### DISCUSSION TITLE
Where do I put libz?(Building on Windows for Windows)

### DISCUSSION THREAD

**Discussion by main-thing** [nillerusr/source-engine#118] [Q&A] [answered]
I have all of the parameters for building. waf.bat cannot find library libz, no matter what I do it can't detect the library. Help would be appreciated.

**Comment by stephen-cusi** [comment]
> 我有构建的所有参数。waf.bat 找不到库 libz，无论我做什么，它都无法检测到该库。帮助将不胜感激。
Same question

**Comment by main-thing** [comment, marked as answer]
Found out after some tests. Need to put it in thirdparty folder.
